src/main/java/org/gitlab4j/api/CommitsApi.java

```diff
@@ -180,6 +180,29 @@ public CommitsApi(GitLabApi gitLabApi) {
         return (new Pager<Commit>(this, Commit.class, itemsPerPage, formData.asMap(),  "projects", projectId, "repository", "commits"));
```

Comment:
I could also modify this method to call the new one that I added, with null for the path, if you would prefer.

Reply:
Done.